### ISSUE TITLE
qMT-SPGR Method Yarnykh is failling

### ISSUE THREAD

**Issue by tanguyduval** [neuropoly/qMRLab#76]
Error:
````
failure: Reference to non-existent field 'FixR1fT2f'.
  SPGR_Y_fun:53 (/Users/Tanguy/code/qMRLab/Models_Functions/SPGRfun/functions/SPGR_Y_fun.m)
  SPGR_SimCurve:58 (/Users/Tanguy/code/qMRLab/Models_Functions/SPGRfun/functions/SPGR_SimCurve.m)
  SPGR.equation:248 (/Users/Tanguy/code/qMRLab/Models/Myelin_Imaging/qMT/SPGR.m)
  test_equation:38 (/Users/Tanguy/code/qMRLab/Test/MoxUnitCompatible/test_Models.m)
````

I improved the testing in branch Testing-all-options.
In this branch all combinations of options are tested. Travis is indeed failling for qMT-SPGR (https://travis-ci.org/neuropoly/qMRLab), and other qMT methods...
Errors revealed by the test should be solved in this branch

**Comment by mathieuboudreau** [comment]
This field used to be set in SPGR_OptionsGUI.m, using a checkbox in the GUI. You can see it in [this line here](https://github.com/neuropoly/qMRLab/blob/ca748715cc4893ccabafa13ec1e7557f9b7b99ce/Models_Functions/SPGRfun/SPGR_OptionsGUI.m#L294).

@tanguyduval you deleted that file on September 8th, in a commit called "add Default Sf Table", [see here](https://github.com/neuropoly/qMRLab/commit/901a8fef88b6e018b3c175d72c519cf307254527).

Was this a mistake?

If not, where should the FixR1fT2f property of the FitOpt struct be set now that we are using the modular model system, when a checkbox is set?

**Comment by tanguyduval** [comment]
Thanks for the investigation @mathieuboudreau 

SPGR_OptionsGUI.m was obsolete with the new modular-based method.  

I just added the FixR1fT2f Box in 16581dceaa367dc53b81906b3a940e066b4539c9
but the behavior could be improved because it should be hidden for SledPike (not used in this case)

**Comment by mathieuboudreau** [comment]
Great, thanks for the quick fix and clarification @tanguyduval !

I agree on the last point, however this was always an issue in the past as well (the option has always been visible for all qMT methods, unfortunately).

A+

**Comment by tanguyduval** [comment]
Ranabi solved in 9dfff9bd8dd2ae459bb33615976609453ff521c0
